.wordpress-org/blueprints/blueprint.json

```
@@ -0,0 +1,24 @@
+{
+  "$schema": "https://playground.wordpress.net/blueprint-schema.json",
+  "landingPage": "\/wp-admin\/tools.php?page=plugin-check",
+  "phpExtensionBundles": [
+    "kitchen-sink"
+  ],
+  "steps": [
+    {
+      "step": "login",
+      "username": "admin",
+      "password": "password"
+    },
+    {
+      "step": "installPlugin",
+      "pluginZipFile": {
+        "resource": "wordpress.org/plugins",
```

```suggestion
        "resource": "wordpress.org\/plugins",
```
Not blocker but for consistency.